app/scripts/modules/core/delivery/executionGroup/execution/execution.directive.js

```diff
@@ -94,6 +94,15 @@ module.exports = angular
       });
     };
 
+    this.restartDetails = () => {
```

Comment:
It shouldn't - the execution itself is basically immutable. When any change is detected, we replace the entire execution object, so Angular will re-render the entire component tree related to it.

The only things that change are elements of the view state, e.g. which, if any details section is expanded.